server/src/main/java/org/opensearch/index/translog/TranslogManager.java

```diff
@@ -131,4 +131,5 @@ public interface TranslogManager {
     Clean up if any needed on deletion of index
      */
     void onDelete();
+
```

Comment:
lets remove unwanted changes .

Reply:
oh.. i thought spotless will take care of it. fixed now.